helix-rest/src/main/java/org/apache/helix/rest/clusterMaintenanceService/MaintenanceManagementService.java

```diff
@@ -67,10 +67,17 @@
 
 
 public class MaintenanceManagementService {
+  private static class MaintenanceExecutorService {
```

Comment:
shouldn't this be configurables?

Reply:
+1, we should have some defaults but this should be configurable when you boot up helix-rest.